### DISCUSSION TITLE
earthworm 组织名征集

### DISCUSSION THREAD

**Discussion by baboon-king** [cuixueshe/earthworm#601] [General]
Github 官方不再释放不活跃的用户名。
github.com/earthworm 没戏了
https://github.com/aermin/blog/issues/44#issuecomment-867303039
<img width="830" alt="image" src="https://github.com/cuixueshe/earthworm/assets/63645337/c3b4a2c5-7576-4980-a68e-e9bd0a58948d">

可以征集一些新的名字

**Comment by baboon-king** [comment]
earthworm-official

**Comment by baboon-king** [comment]
earthworm-9527

**Comment by baboon-king** [comment]
earthworm001

**Comment by shellingfordly** [comment]
earthworm-online

**Comment by shellingfordly** [comment]
earthworms

**Comment by cwandev** [comment]
不少组织名是根据域名+域名后缀决定的。

earthworm.fun/earthworm-fun
earthworm.io/earthworm-io
earthworm.zone/earthworm-zone
...

保持一致性的名称，更容易使人们记住。
